src/bin_editor/Oni2_editor.re

```diff
@@ -133,15 +133,15 @@ if (cliOptions.syntaxHighlightService) {
       isDirty := true;
     };
 
-    let _: unit => unit =
-      Tick.interval(
-        _dt =>
-          if (isDirty^) {
-            update(<Root state=currentState^ />);
-            isDirty := false;
-          },
-        Time.seconds(0),
-      );
+    let tick = _dt => {
+      let _: bool = Luv.Loop.run(~mode=`NOWAIT, ());
```

Comment:
This is all the wiring needed to integrate Luv into the app loop.

Reply:
Cool! Very straightforward 👍